### ISSUE TITLE
updatebot-prod does not have permissions to change assignee

### ISSUE THREAD

**Issue by metalcanine** [mozilla-services/updatebot#181]
```
Exception: {
  'message': 'You tried to change the Assignee field from nobody@mozilla.org to krosylight@mozilla.com , but only the assignee of the bug, or a user with the required permissions may change that field. Additonal information:The bug reporter is not allowed to change this field If you are attempting to confirm an unconfirmed bug or edit the fields of a bug, find out how to get the necessary permissions.',
  'documentation': 'https://bmo.readthedocs.io/en/latest/api/',
  'error': True,
  'code': 115
}
```

maybe we could get around this by initially setting updatebot as the assignee, or finding the permissions needed for the account we're using in production and grant it the necessary permissions

**Comment by tomrittervg** [comment]
I asked in #bmo for dkl to assign the perms

**Comment by metalcanine** [comment]
this was solved by dkl with [1711982](https://bugzilla.mozilla.org/show_bug.cgi?id=1711982)
